api_usage.md

```diff
@@ -0,0 +1,254 @@
+# Guide on ingesting and publishing data to the VEDA data store & STAC API
```

Comment:
We have previously made an effort to move data ingestion docs and VEDA-specific things out of the repos and into the docs here: https://nasa-impact.github.io/veda-docs/contributing/dataset-ingestion/

I think we should rather stick to that and link to the docs from the repos, if need be.

Reply:
Thanks for the heads up; I'll get things updated this morning